libpromises/crypto.c

```diff
@@ -502,10 +502,29 @@ size_t CipherTextLengthMax(const EVP_CIPHER* cipher, size_t plaintext_length)
     return plaintext_length + (CipherBlockSizeBytes(cipher) * 2) - 1;
 }
 
+size_t PlainTextLengthMax(const EVP_CIPHER* cipher, size_t ciphertext_length)
+{
+    if(ciphertext_length <= 0)
```

Comment:
Can't be negative, so I would change this to `assert(ciphertext_length == 0)`. BTW why can't it be zero?